alkira/provider.go

```diff
@@ -33,46 +33,50 @@ func Provider() *schema.Provider {
-			"alkira_credential_pan_instance":      resourceAlkiraCredentialPanInstance(),
-			"alkira_cloudvisor_account":           resourceAlkiraCloudVisorAccount(),
-			"alkira_group_connector":              resourceAlkiraGroupConnector(),
-			"alkira_group":                        resourceAlkiraGroup(),
-			"alkira_group_user":                   resourceAlkiraGroupUser(),
-			"alkira_list_as_path":                 resourceAlkiraListAsPath(),
-			"alkira_list_community":               resourceAlkiraListCommunity(),
-			"alkira_list_extended_community":      resourceAlkiraListExtendedCommunity(),
-			"alkira_list_global_cidr":             resourceAlkiraListGlobalCidr(),
-			"alkira_internet_application":         resourceAlkiraInternetApplication(),
-			"alkira_policy":                       resourceAlkiraPolicy(),
-			"alkira_policy_nat":                   resourceAlkiraPolicyNat(),
-			"alkira_policy_nat_rule":              resourceAlkiraPolicyNatRule(),
-			"alkira_policy_prefix_list":           resourceAlkiraPolicyPrefixList(),
-			"alkira_policy_rule":                  resourceAlkiraPolicyRule(),
-			"alkira_policy_rule_list":             resourceAlkiraPolicyRuleList(),
-			"alkira_segment":                      resourceAlkiraSegment(),
-			"alkira_segment_resource":             resourceAlkiraSegmentResource(),
-			"alkira_segment_resource_share":       resourceAlkiraSegmentResourceShare(),
-			"alkira_service_fortinet":             resourceAlkiraServiceFortinet(),
-			"alkira_service_pan":                  resourceAlkiraServicePan(),
-			"alkira_tenant_network":               resourceAlkiraTenantNetwork(),
+			"alkira_billing_tag":                             resourceAlkiraBillingTag(),
+			"alkira_byoip_prefix":                            resourceAlkiraByoipPrefix(),
+			"alkira_connector_akamai_prolexic":               resourceAlkiraConnectorAkamaiProlexic(),
+			"alkira_connector_aws_vpc":                       resourceAlkiraConnectorAwsVpc(),
+			"alkira_connector_azure_vnet":                    resourceAlkiraConnectorAzureVnet(),
+			"alkira_connector_cisco_sdwan":                   resourceAlkiraConnectorCiscoSdwan(),
+			"alkira_connector_gcp_vpc":                       resourceAlkiraConnectorGcpVpc(),
+			"alkira_connector_oci_vcn":                       resourceAlkiraConnectorOciVcn(),
+			"alkira_connector_internet_exit":                 resourceAlkiraConnectorInternetExit(),
+			"alkira_connector_ipsec":                         resourceAlkiraConnectorIPSec(),
+			"alkira_credential_aws_vpc":                      resourceAlkiraCredentialAwsVpc(),
+			"alkira_credential_azure_vnet":                   resourceAlkiraCredentialAzureVnet(),
+			"alkira_credential_cisco_sdwan":                  resourceAlkiraCredentialCiscoSdwan(),
+			"alkira_credential_checkpoint":                   resourceAlkiraCredentialCheckpoint(),
+			"alkira_credential_checkpoint_instance":          resourceAlkiraCredentialCheckpointInstance(),
```

Comment:
I guess those two resources are internally managed and will be deleted automatically? Do we want to manage those credentials as separate resource here?